server/src/main/java/org/candlepin/audit/EventSinkImpl.java

```diff
@@ -196,9 +208,12 @@ public void queueEvent(Event event) {
      */
     @Override
     public void sendEvents() {
+        log.debug("Committing ActiveMQ transaction");
+
         try {
-            log.debug("Committing ActiveMQ transaction.");
-            getClientSession().commit();
+            ClientSession session = this.getClientSession();
+            session.commit();
+            session.close();
```

Comment:
Put the close() into a finally block so that an attempt is always made to close the session, even on commit failure. If closing the session fails, there's not really anything we can do -- log and move on.